controllers/apply/awards-for-all/form.test.js

```diff
@@ -1025,6 +1025,37 @@ describe('Contacts', () => {
             );
         }
     );
+
+    test.each([
+        'mainContactLanguagePreference',
+        'seniorContactLanguagePreference'
+    ])('%p must exist and be a valid choice for Wales', function(fieldName) {
+        assertValidByKey({
+            projectCountry: 'england',
+            [fieldName]: 'wales'
```

Comment:
This test looks a bit wonky? Should this be `null` or something else?

Reply:
Oops this was me debugging a test, sorry! Will fix.